package.json

```diff
@@ -15,9 +15,9 @@
     "express-history-api-fallback": "2.2.1",
     "highland": "2.13.0",
     "html-minifier": "3.5.19",
-    "minimalcss": "0.7.6",
+    "minimalcss": "0.7.10",
     "mkdirp": "0.5.1",
-    "puppeteer": "^1.4.0",
+    "puppeteer": "^1.8.0",
```

Comment:
puppeteer version suppose to be the same as "minimalcss" otherwise npm/yarn will download two copies of puppeteer (which is a lot)